src/main/resources/db/migration/V1.11__insert_admin.sql

```diff
@@ -0,0 +1,2 @@
+insert into users (first_name, last_name, email_address, password, role)
+VALUES ('Admin', 'Admin', 'admin@gmail.com', '$2a$10$oQAVxZpVOffbRJ1XyY9eM.aTHoezWY1lL/a5fcZWsZVLmyoUHxCrO', 1);
```

Comment:
I hope this is bcrypt encryption, right?

Reply:
Yes it is